Kitodo/src/main/java/de/sub/goobi/helper/WebDav.java

```diff
@@ -62,9 +62,14 @@ public WebDav() {
         FilenameFilter filter = new FileNameEndsWithFilter("]");
 
         try {
-            URI directoryName = serviceManager.getUserService().getHomeDirectory(currentUser).resolve(inVerzeichnis);
-            files = fileService.getSubUris(filter, directoryName);
-        } catch (Exception ioe) {
+            if (currentUser != null) {
+                URI directoryName = serviceManager.getUserService().getHomeDirectory(currentUser).resolve(inVerzeichnis);
+                files = fileService.getSubUris(filter, directoryName);
+            } else {
+                Helper.setFehlerMeldung("uploadFromHomeAlle abgebrochen, Fehler - no user assigned");
```

Comment:
Please, do not mix German and English. Please use one language or use a translation key for translatable error messages.

Reply:
Is there a key for such a message in German or better whole message in English?

Reply:
No, this must be created for both languages as you added this in this pull request.